### PR TITLE
update polling message

### PR DESCRIPTION
**Pull request by TalGumi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by jlevypaloalto** [comment]
Hi @TalGumi, thank you for contributing!

Can you please add in the description the purpose of this change?

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again!

File: Packs/ThreatGrid/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "Cisco Secure Malware Analytics",
     "description": "Query and upload samples to Cisco threat grid.",
     "support": "xsoar",
-    "currentVersion": "2.0.22",
+    "currentVersion": "2.1.0",
```
Comment (BEAdi) [inline Packs/ThreatGrid/pack_metadata.json:5]:
```suggestion
    "currentVersion": "2.0.23",
```
I think this should be a minor change.

File: Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.py
Comment (BEAdi) [inline Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.py:1]: Can you revert the changes that are not related to your edit? For a better git log.

**Review by BEAdi — APPROVED** [review]
Great job!

File: Packs/ThreatGrid/ReleaseNotes/2_1_0.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Cisco Secure Malware Analytics (Threat Grid) v2
+
+- Added sample ID value to the human-readable for the ***threat-grid-sample-upload*** command.
```
Comment (ShirleyDenkberg) [inline Packs/ThreatGrid/ReleaseNotes/2_1_0.md:6]:
```suggestion
Added a sample ID value to the human-readable output for the ***threat-grid-sample-upload*** command.
```

File: Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.py [hunk 3]
```diff
@@ -1006,8 +998,9 @@ def schedule_command(args: dict[str, Any], client: Client) -> PollResult:
     Returns:
         ScheduledCommand: Command, args, timeout and interval for CommandResults.
     """
-
+    first_run = False
```
Comment (BEAdi) [inline Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.py:1001]: You can add the `hide_polling_output` argument in the yml file that does this.
Look into the `polling_function` decorator to see it, and you can see an example in `CrowdStrikeFalcon`.
More details here: https://xsoar.pan.dev/docs/integrations/scheduled-commands#the-pollresult-class:~:text=to%20Ignore%20Scheduled%20War%20Room%20Entries%20(as%20indicated%20below)%20add%20hide_polling_output%20as%20a%20boolean%20argument%20to%20the%20command%20in%20the%20yml%20file.

File: Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.py
Comment (TalGumi) [reply Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.py:1]: Do you mean the format changes?

Comment (BEAdi) [reply Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.py:1]: Yes
